### PR TITLE
Wrapped any iterpolated string with FormattableString.Invariant(), so…

### PR DESCRIPTION
**Pull request by jeffrey-opdam**
… it uses the Invariant cultureinfo.

Thank you for your pull request!
By completing this pull request, you agree to the [Contributing License Agreement](https://github.com/ALM-Rangers/Team-Services-Bot/blob/master/.github/CLA.md).

Fixes # .

Changes proposed in this pull request:  
- Wrapped any iterpolated string with FormattableString.Invariant
- Enabled CA1305
- 

@ALM-Rangers/Team-Services-Bot

### CODE REVIEW

File: src/Team-Services-Bot.Api/Cards/LogOnCard.cs [hunk 1]
```diff
@@ -39,7 +40,7 @@ public LogOnCard(string appId, Uri authorizeUrl, string channelId, string text,
         {
             var button = new CardAction
             {
-                Value = string.Format(UrlOAuth, appId, channelId, userId, Scope, authorizeUrl),
+                Value = string.Format(CultureInfo.InvariantCulture, UrlOAuth, appId, channelId, userId, Scope, authorizeUrl),
```
Comment (rjarrattmsft) [inline src/Team-Services-Bot.Api/Cards/LogOnCard.cs]: Why not use FormattableString here as well? Must say I didn't know about FormattableString.

File: src/Team-Services-Bot.Api.UnitTests/Controllers/AuthorizeControllerTests.cs [hunk 2]
```diff
@@ -44,7 +44,6 @@ public class AuthorizeControllerTests
 
             var controller = new AuthorizeController(
                 botService.Object,
-                new TelemetryClient(),
```
Comment (Hussar77) [inline src/Team-Services-Bot.Api.UnitTests/Controllers/AuthorizeControllerTests.cs]: There is "using Microsoft.ApplicationInsights;" which should be removed as well.

Comment (jeffrey-opdam) [inline src/Team-Services-Bot.Api.UnitTests/Controllers/AuthorizeControllerTests.cs]: As it is here a const, which means you have to use string.format.

File: src/Team-Services-Bot.Api/Cards/LogOnCard.cs (hunk 1 above)
Comment (jeffrey-opdam) [inline src/Team-Services-Bot.Api/Cards/LogOnCard.cs]: As it is here a const, which means you have to use string.format.

**Pull request merged**